### ISSUE TITLE
Logrotate needed on Hadoop workers for /logs directory

### ISSUE THREAD

**Issue by stvoutsin** [wfau/aglais#53]
The Hadoop logs directory seems to grow quite fast, which we discovered recently after out-of-space errors started appearing when trying to run Spark jobs. The cluster recovered after clearing the logs from each worker.
We need to setup Logrotate on the Hadoop workers for /logs directory.

**Comment by Zarquan** [comment]
Do we have notes on how to detect and fix the problem ?

**Comment by stvoutsin** [comment]
Some notes on this issue can be found here: https://github.com/stvoutsin/aglais/blob/d76a516d3e5dc35b8eb4cd8ee21ec7bc4cb618cb/notes/stv/20200124-hdfs-disk-space-issue.txt

I've created a branch to work on this

**Comment by Zarquan** [comment]
@stvoutsin Is this still a problem ?
How do we track this in other deployments ?

**Comment by stvoutsin** [comment]
@Zarquan Yep, this is still an outstanding issue. We need to check whether there is a configuration we can set in hadoop to handle this, otherwise we can create a cron job to rotate + zip the log files on each node.

**Comment by Zarquan** [comment]
Need to make this reproducible and testable
1. Run a test job from the command line and capture output messages.
1. Temporarily use up all the disc space on the worker nodes.
1. Run a test job from the command line and capture output messages.
1. Free up disc space on the worker nodes.
1. Run a test job from the command line and capture output messages.
1. Use jq/sed/awk/grep or similar to show change in the output messages before/during/after

**Comment by Zarquan** [comment]
Is this still an issue ?

**Comment by Zarquan** [comment]
I've added this to DR3 milestone because we should check if this is an issue before we try to support lots of users.
